docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+            "max_block_net_usage": 1048576,
+            "target_block_net_usage_pct": 1000,
+            "max_transaction_net_usage": 524288,
+            "base_per_transaction_net_usage": 12,
+            "net_usage_leeway": 500,
+            "context_free_discount_net_usage_num": 20,
+            "context_free_discount_net_usage_den": 100,
+            "max_block_cpu_usage": 400000,
+            "target_block_cpu_usage_pct": 1000,
+            "max_transaction_cpu_usage": 300000,
+            "min_transaction_cpu_usage": 100,
+            "ultra_veto_enabled": 1,
+            "max_transaction_lifetime": 3600,
+            "deferred_trx_expiration_window": 600,
+            "max_transaction_delay": 3888000,
+            "max_inline_action_size": 524287,
+            "max_inline_action_depth": 10,
+            "max_authority_depth": 10
+          }
+        }
+merger:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    persistentVolumeClaimEnabled: false
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+relayer:
+  replicaCount: 1
+  deployment:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    relayerSource2: ""
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+substreams-tier1:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    substreamsStateStoreUrl: "file:///storage/state-store"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+substreams-tier2:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    substreamsStateStoreUrl: "file:///storage/state-store"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+firehose:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+
+```
+
+Some important configurations you need to know:
+
+- `statefulset: enabledPVC = false`: This will disable the persistent volume use for Ultra's deployment
+- `localVolume: enabled = true`: This will enable local storage to replace cloud storage, this will make use of `minikube` persist file store. You can read more here [Persistent Volumes](https://minikube.sigs.k8s.io/docs/handbook/persistent_volumes/).
```

Comment:
`You can read more here [Persistent Volumes](https://minikube.sigs.k8s.io/docs/handbook/persistent_volumes/).` should be before the above line 296, because the term persistent volume is used in the line.

Reply:
I updated the above like it's about Persistent Volume Claim instead, and that will only be used in Ultra's deployment.